frontend/packages/noobaa-storage-plugin/src/components/activity-card/data-resiliency-activity.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Progress, ProgressSize } from '@patternfly/react-core';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import { getResiliencyProgress } from '../../utils';
+
+export const isDataResiliencyActivity = (results: PrometheusResponse): boolean => {
+  const progress = getResiliencyProgress(results);
+  return progress && progress < 100;
```

Comment:
```suggestion
  return progress < 100;
```
progress can only be number or NaN. No need to guard it